cmake/modules/SwiftSupport.cmake

```diff
@@ -0,0 +1,75 @@
+
+include(CMakeParseArguments)
+
+function(add_swift_library library)
```

Comment:
It would be super keen if this was maintained by the swift team and/or a standard cmake thing

Reply:
I would absolutely love that too.  Newer versions of CMake have introduced some support for swift, and I could see this going away in the future.